server/graphql/v1/mutations/collectives.js

```diff
@@ -177,6 +178,74 @@ export async function createCollective(_, args, req) {
   return collective;
 }
 
+export async function createCollectiveFromGithub(_, args, req) {
+  if (!req.remoteUser) {
+    throw new errors.Unauthorized({
+      message: 'You need to be logged in to create a collective',
+    });
+  }
+
+  if (!args.collective.name) {
+    throw new errors.ValidationFailed({ message: 'collective.name required' });
+  }
+
+  let collective;
+  const collectiveData = { ...args.collective };
```

Comment:
There is a lot of extra functionnality that needs to be ported from here:
https://github.com/opencollective/opencollective-website/blob/04d62f7047cb4aa25946ab226840b6203a16a027/frontend/src/containers/OnBoarding.js#L108

- default tiers
- repository settings
- repository website
- repository tags

Reply:
I will add this tonight.